noxfiles/docs_nox.py

```diff
@@ -33,7 +33,7 @@ def docs_serve(session: nox.Session) -> None:
         "docs",
         "/bin/bash",
         "-c",
-        "pip install -e /fides && mkdocs serve --dev-addr=0.0.0.0:8000",
```

Comment:
the reason this was in there is so that auto-generated docs (like the CLI and API) would match the local state and be verifiable

Reply:
looks like you already solved this in the dockerfile though, that's a much better solution